### DISCUSSION TITLE
Configuration tool

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [pmndrs/react-spring#1525] [Support]
Hi! Do you know if there's a configuration tool with all the settings, in order to be able to quickly test the animation result? The only i know is https://react-spring-visualizer.com/ but it only has few settings to tweak.

**Comment by joshuaellis** [comment]
Not sure what you mean by settings, so you mean config?

Check the docs, I made a demo with a fair amount of variables using leva

**Reply by Emiliano-Bucci** [reply]
Oh, i didn't knew it! Thanks for the info! :)
